### ISSUE TITLE
Cannot use RunPluginVerifierTask.FailureLevel.ALL from Groovy build.gradle

### ISSUE THREAD

**Issue by TWiStErRob** [JetBrains/gradle-intellij-plugin#1323]
**Describe the bug**
As title.

**To Reproduce**
1. Clone https://github.com/TWiStErRob/repros/tree/master/idea/gradle-intellij-plugin-FailureLevel-ALL-groovy
2. `gradlew runPluginVerifier`

**Expected behavior**
Complies and executes.

**Actual behavior**
```
> No signature of method: static org.jetbrains.intellij.tasks.RunPluginVerifierTask$FailureLevel$Companion.getALL() is applicable for argument types: () values: []
  Possible solutions: getALL(), getAt(java.lang.String), getNONE(), putAt(java.lang.String, java.lang.Object), getClass(), each(groovy.lang.Closure)
```
This is because `RunPluginVerifierTask.FailureLevel.Companion` is a class object, not referencing the static field `FailureLevel.Companion`.

**Environment:**
 - OS: Windows
 - Gradle IntelliJ Plugin Version 1.13.0
 - Gradle Version 8.0

**Additional context**
I tried many-many variants, but none of them worked:
```
failureLevel = RunPluginVerifierTask.FailureLevel.Companion.getALL()
failureLevel = RunPluginVerifierTask.FailureLevel.Companion.ALL
failureLevel = RunPluginVerifierTask.FailureLevel.ALL
failureLevel = RunPluginVerifierTask.FailureLevel.Companion.INSTANCE.ALL
```

**Workaround**
```
//noinspection UnnecessaryQualifiedReference
	failureLevel = EnumSet.allOf(org.jetbrains.intellij.tasks.RunPluginVerifierTask.FailureLevel.class)
```

**Comment by hsz** [comment]
This is odd, but actually, the `failureLevel = RunPluginVerifierTask.FailureLevel.ALL` variant works for me, however IDE reports `Access to 'ALL' exceeds its access rights`.

**Comment by hsz** [comment]
Fixed with 6b25f1b — it's available in `1.13.1-SNAPSHOT`.
